example/61_conv_fwd_activ/convnd_fwd_xdl_clippedrelu_fp16.cpp

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#include "convnd_fwd_activ_common.hpp"
+
+using OutElementOp = ck::tensor_operation::element_wise::ClippedRelu;
+
+using DeviceGroupedConvNDFwdActivInstance = DeviceGroupedConvNDFwdInstance<OutElementOp>;
+#include "run_convnd_fwd_activ_example.inc"
+
+int main(int argc, char* argv[]) { return run_convnd_fwd_example(argc, argv) ? 0 : 1; }
```

Comment:
I guess this implicit conversion from bool to int is pretty common in our codebase, no need for ternary here
```suggestion
int main(int argc, char* argv[]) { return !run_convnd_fwd_example(argc, argv); }
```